igvc2023/params/foot_print.yaml

```
@@ -1 +1 @@
-footprint: [ [0.2, 0.33], [0.2, -0.33], [-0.65, -0.33], [-0.65, 0.33] ]
+footprint: [ [0.2, 0.35], [0.2, -0.35], [-1.4, -0.29], [-1.4, 0.29] ]
```

0.35はどのように計算しましたか？
以下の計算式だと思いますが違いますか？

(0.58+0.047)/2=0.3135≃0.32
https://github.com/KBKN-Autonomous-Robotics-Lab/IGVC2023-src/blob/f9646c20ad7619abf9f7e0d4c1797bc57a0f209b/igvc2023/urdf/orange.xacro#L152
```suggestion
footprint: [ [0.2, 0.32], [0.2, -0.32], [-1.4, -0.32], [-1.4, 0.32] ]
```
0.35はどのように計算しましたか？